cli/parse_pdfs.py

```diff
@@ -510,11 +556,13 @@ def get_model(
         )
 
     # FIXME: handle EmptyFileError here using _pdf_num_pages
```

Comment:
This fixme doesn't look like it refers to the model function below?

Can we delete it as it's causing the github actions to fail as the fixme is in there?